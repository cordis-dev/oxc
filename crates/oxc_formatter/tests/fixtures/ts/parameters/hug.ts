 const assertFilteringFor = (expected: {
   [T in TestFilterTerm]?: boolean;
 }) => {};
 