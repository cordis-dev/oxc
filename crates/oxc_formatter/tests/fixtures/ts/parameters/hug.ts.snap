---
source: crates/oxc_formatter/tests/fixtures/mod.rs
---
==================== Input ====================
 const assertFilteringFor = (expected: {
   [T in TestFilterTerm]?: boolean;
 }) => {};
 
==================== Output ====================
const assertFilteringFor = (expected: {
  [T in TestFilterTerm]?: boolean;
}) => {};

===================== End =====================
